packages/pill/src/index.js

```diff
@@ -0,0 +1,51 @@
+// vendor modules
+import React from 'react';
+import PropTypes from 'prop-types';
+import getClassNames from 'classnames';
+import _ from 'lodash';
+// css modules
+import './index.scss';
+
+/**
+ * @example ../README.md
+ */
+export default class Pill extends React.Component {
+  static propTypes = {
+    className: PropTypes.string,
+    variant: PropTypes.oneOf(['outlined', 'raised']),
```

Comment:
I don't think this component is as per the design. zpl://screen?sid=5e77a90462b4676fb1ec9e7f&pid=5df72d1e602c331cb8dc8b79

I have tagged you in the design for various variants. I believe this component complete requires rework.